src/VisualStudio/Core/Def/Implementation/CommandBindings.cs

```diff
@@ -7,6 +7,10 @@ namespace Microsoft.VisualStudio.Editor.Implementation
 {
     internal sealed class CommandBindings
     {
+        [Export]
+        [CommandBinding(Guids.RoslynGroupIdString, ID.RoslynCommands.GoToBase, typeof(GoToBaseCommandArgs))]
```

Comment:
As mentioned earlier, any changes in this file shouldn't be necessary if the command is implemented on the editor side.